tc/sgx/trusted_worker_manager/enclave/work_order_enclave.cpp

```diff
@@ -33,6 +33,8 @@ ByteArray last_serialized_response;
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 tcf_err_t ecall_HandleWorkOrderRequest(const uint8_t* inSerializedRequest,
     size_t inSerializedRequestSize,
+    const uint8_t* inWorkOrderExtData,
+    size_t inWorkOrderExtDataSize,
```

Comment:
Is this file specific to singleton ? Then please add a check to ensure these values passed are NULL

Reply:
Ok